app/models/reservation.rb

```diff
@@ -0,0 +1,12 @@
+class Reservation < ApplicationRecord
+  belongs_to :user
+  belongs_to :event
+  belongs_to :skill
+  belongs_to :reservation_status
+  has_many :driven_pairs, class_name: 'Pair', foreign_key: 'person_1'
+  has_many :navigated_pairs, class_name: 'Pair', foreign_key: 'person_2'
+
+  def pairs
+    driven_pairs + navigated_pairs
```

Comment:
Reservation is actually a representation of a User and its state at an Event. It also includes stuff like whether a User 
- has RSVP'd
- is ready for pairing
- or canceled